doc/man/vegastrike-engine.1

```diff
@@ -27,11 +33,20 @@ vegasettings
 .PP
 Program to create and edit the settings for Vega Strike.
 .SS
-vegastrike
+vegastrike-engine
 .PP
 Launch Vega Strike and allows the user to select a campaign to run or load a saved game.
+This uses the legacy behavior of defaulting to the local directory for the game assets.
 .PP
 Saved games must be saved at a base or planet.
+.SS
+vegastrike
+.PP
+Same as
+.ft I
+vegastrike-engine
+.ft P
+except that it uses the legacy behavior of defaulting to the local directory for the game assets.
```

Comment:
I'm confused. Aren't we repeating ourselves here? ;-)

Reply:
will clarify to be less confusing and more accurate.